code/modules/events/antag_ghost_respawn.dm

```diff
@@ -334,22 +334,19 @@
 						failed = 1
 
 				if ("Changeling")
-					var/mob/living/R2 = M3.humanize()
-					if (R2 && istype(R2))
-						M3 = R2
-						R2.make_changeling()
-						role = ROLE_CHANGELING
-						objective_path = /datum/objective_set/changeling
+					var/mob/living/L = M3.humanize()
```

Comment:
Looks like this is missing any kind of send to arrivals code, currently will spawn the changeling wherever their ghost is.

Reply:
The code pertaining to moving the chosen mob to Arrivals may be found in the switch statement on line 401; however it runtimes during the `set_loc()` proc due to `M3` not being updated correctly in the code above, thus preventing the mob from being sent to Arrivals. The same is true for the vampire datumisation PR. Both have been amended, alongside fixing the same issue in the arcfiend switch statement branch.